manifests/c/CompLabs/GoogleDocs/2022.12.1/CompLabs.GoogleDocs.locale.en-US.yaml

```
@@ -0,0 +1,17 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: CompLabs.GoogleDocs
+PackageVersion: 2022.12.1
+PackageLocale: en-US
+Publisher: Comp Labs
+PublisherUrl: https://complabs.in/
+PublisherSupportUrl: https://github.com/Comp-Labs/Google-Docs/issues
+PrivacyUrl: https://complabs.in/legal/privacy-policy
+PackageName: Unofficial Google Docs
+PackageUrl: https://complabs.in/apps/google-docs
```

```suggestion
PackageUrl: https://github.com/Comp-Labs/Google-Docs
```

[Project URL](https://techfiddle.io/apps/google-docs) on their [GitHub Repo](https://github.com/Comp-Labs/Google-Docs) is also `HTTP 404 Not Found`.